mantle/kola/tests/podman/podman.go

```diff
@@ -290,7 +290,7 @@ func podmanNetworksReliably(c cluster.TestCluster) {
 
 	numPass := strings.Count(string(output), "PASS")
 
-	if numPass != 100 {
-		c.Fatalf("Expected 100 passes, but output was: %s", output)
+	if numPass <= 98 {
```

Comment:
I'm not sure if arithmetic comparison works like this in bash. See https://stackoverflow.com/a/18668580

Reply:
oh snap.. ignore my previous comment. I was in the "external tests" frame of mind and didn't realize this was in `mantle`.